src/corelibs/U2View/src/ov_sequence/DetViewSequenceEditor.cpp

```diff
@@ -343,8 +342,10 @@ void DetViewSequenceEditor::sl_editMode(bool active) {
     }
 }
 
-void DetViewSequenceEditor::sl_changeCursorColor() {
-    cursorColor = (cursorColor == QColor(Qt::black)) ? Qt::darkGray : Qt::black;
+void DetViewSequenceEditor::sl_cursorAnimationTimerCallback() {
+    // Reproduce 'blink' effect: change the cursor color periodically.
+    // Show 'edit-cursor' only for the focused view and hide it (use transparent color) otherwise.
+    cursorColor = view->hasFocus() ? (cursorColor == Qt::black ? Qt::darkGray : Qt::black) : Qt::transparent;
```

Comment:
I think, there is no need to update view on each callback if color `cursorColor` is black and the view doesn't have focus. You can add something like this in the beginning:
```
if (!view->hasFocus() && cursorColor == Qt::black) {
    return;
}
```

Reply:
Agree, this is a valid optimization. Fixed.